testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -67,7 +67,7 @@
     SUPPORTS_RENAME_MATERIALIZED_VIEW_ACROSS_SCHEMAS(SUPPORTS_RENAME_MATERIALIZED_VIEW),
 
     SUPPORTS_INSERT,
-    SUPPORTS_INSERT_NOT_NULL_COLUMN(SUPPORTS_INSERT),
+    SUPPORTS_NOT_NULL_CONSTRAINT,
```

Comment:
default it from SUPPORTS_CREATE_TABLE. 
SUPPORTS_NOT_NULL_CONSTRAINT cannot be supported without SUPPORTS_CREATE_TABLE